lms/djangoapps/onboarding_survey/views.py

```diff
@@ -284,6 +284,7 @@ def organization(request):
     context['is_poc'] = extended_profile.is_poc
     context['is_first_user'] = is_first_signup_in_org(extended_profile.organization)
     context['organization_name'] = extended_profile.organization.name
+    context['google_place_api_key'] = settings.GOOGLE_PLACE_API_KEY
```

Comment:
Why are you sending it via context. Are we able to get GOOGLE_PLACE_API_KEY directly from settings inside template?

Reply:
It's a django template not mako and we cannot access settings inside that template directly.

Reply:
ok that's fine then